empirical-org/Empirical-Core
# final accessibility revisions for connect and diagnostic

…terfaces fully navigable using keyboard (#5813)

* add backup fonts

* Auto stash before cherrypick of "add backup fonts"

* add new progress bar to diagnostic

* add new progress bar to connect

* wip

* bump qcl version

* install new qcl and make additional minor css changes

* add new progress bar to proofreader

* bump qcl version again

* update qcl version

* make it so that the progress bar shows up on titlecard slides

* add new focus styles and apply them to connect navbar

* apply new focus and tabindexes to quill-buttons in connect and diagnostic

* put new focus classes in qcl

* version bump qcl

* try defining tabIndex differently in qcl to get typecript to calm down

* wip

* patch qcl

* re-install qcl

* wip

* wip

* bump qcl

* minor fix

* specify stateless component

* pass inner props instead of all props

* one more try to get htmlDropdownOption around tsc

* install latest rollup

* install latest qcl

* minor change

* patch qcl

* install updated qcl

* get dropdown working for custom html stuff

* patch qcl

* patch qcl again

* try to get other types of dropdowns working with keyboard navigation

* patch qcl

* wip

* fix some weird behavior with other dropdown inputs

* patch qcl again

* make sure other kinds of dropdown are fully functional

* patch version again

* install qcl where necessary

* wip

* update fillInBlank component to work more seamlessly

* update snapshots

* dropdown revisions

* bump minor version

* wip

* patch qcl

* wip

* point ell diagnostic back to correct path

* constantize mouse and key down

* don't hide overflow

* connect and diagnostic accessibility odds and ends - approved by Tom (#5814)

* fix merge conflict

* put aria labels back

* patch qcl

* install the qcl changes

* update snapshot

* use student root component to hide or show focus states

* wip

* wip

* patch qcl version

* update qcl everywhere necessary

* have connect use the same contenteditable setup as comprehension

* wip

* wip

* patch

* fix animations

* update edit caret positioning to not use dynamic exports

## WHAT

## WHY

## HOW

## Screenshots
(If applicable. Also, please censor any sensitive data)

## Have you added and/or updated tests?
(The answer should mostly be 'YES'. If you answer 'NO', please justify.)
